shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/repositories/ErrorBannerStateRepository.kt

```diff
@@ -14,13 +14,16 @@ protected constructor(initialState: ErrorBannerState? = null) {
     val state = flow.asStateFlow()
 
     private var predictionsStale: ErrorBannerState.StalePredictions? = null
+    private val dataErrors = mutableMapOf<String, ErrorBannerState.DataError>()
 
     protected open fun updateState() {
```

Comment:
This does not call the refresh actions for each error; since the flow can only contain one error at a time, this synthesizes a new `DataError` with an action that calls each error's action. Added a comment.